frontend/tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "jsx": "react-jsx",
     "downlevelIteration": true,
     "noFallthroughCasesInSwitch": true,
-    "typeRoots": ["src/types", "node_modules/@types"]
+    "typeRoots": ["src/typings", "node_modules/@types"]
```

Comment:
Hmm, what is this?

Reply:
Prior to this PR we had TWO folder with custom typings: types and typings. Moved everything to the typings folder and fixed the json config

Reply:
Or we didn't have the src/types folder - can't remember which

Reply:
We had both as per this screenshot

![image](https://user-images.githubusercontent.com/24322224/148624971-85e9a713-90fd-43e8-b11d-75acd0ebfe37.png)